### DISCUSSION TITLE
Support for Multiple Function Calls in Azure OpenAI Function Calling API

### DISCUSSION THREAD

**Discussion by tenthe** [spring-projects/spring-ai#588] [Q&A]
Hi,

I've been using the Azure OpenAI function calling API successfully for single function calls. However, I've encountered an issue when attempting to execute multiple functions sequentially. The API seems to support only one function call per request, leading to a prompt indicating that further functionality is not available.

My use case requires the output from one function to serve as input for the subsequent one. Is this limitation inherent to the API, or am I overlooking something in my implementation?

Thanks for your help,
Philipp

**Comment by tha2015** [comment]
Is this the same issue https://github.com/spring-projects/spring-ai/issues/613 ?

**Reply by tenthe** [reply]
Hi @tha2015,
Thank you very much. 
Yes, this is the same problem. I didn't see the issue, so I'm closing this discussion.

Thank you for pointing it out.
